packages/npm/@amazeelabs/publisher-ui/src/components/Info.tsx

```diff
@@ -5,12 +5,31 @@ import { Dialog, Disclosure, Transition } from '@headlessui/react';
 import { bind } from '@react-rxjs/core';
 import clsx from 'clsx';
 import React, { ComponentProps, Fragment, useState } from 'react';
+import { flushSync } from 'react-dom';
 import { LazyLog } from 'react-lazylog';
 import { ajax } from 'rxjs/ajax';
 
 import { createWebsocketUrl, useStatus } from '../utils/status';
 import Collapsible from './Collapsible';
 
+// Disable React batched updates to fix
+// https://github.com/melloware/react-logviewer/pull/22 without moving from
```

Comment:
Why don't we move to `@melloware/react-logviewer` right away?

Reply:
Never mind, i should read everything.

Reply:
Nope, they merged it :D Lets go! https://github.com/melloware/react-logviewer/pull/22

Reply:
You still should read everything 😛 

https://github.com/melloware/react-logviewer/issues/14 is still opened.

Reply:
Autoscroll is broken in the fork, but still works in the Mozilla package.